deepchem/models/torch_models/weavemodel_pytorch.py

```diff
@@ -0,0 +1,593 @@
+                n_pair_output_feat=n_pair_next,
+                batch_normalize=batch_normalize)
+            nn.init.trunc_normal_(weave_layer.W_AA,
+                                  0,
+                                  std=conv_weight_init_stddevs[ind])
+            nn.init.trunc_normal_(weave_layer.W_PA,
+                                  0,
+                                  std=conv_weight_init_stddevs[ind])
+            nn.init.trunc_normal_(weave_layer.W_A,
+                                  0,
+                                  std=conv_weight_init_stddevs[ind])
+            if weave_layer.update_pair:
+                nn.init.trunc_normal_(weave_layer.W_AP,
+                                      0,
+                                      std=conv_weight_init_stddevs[ind])
+                nn.init.trunc_normal_(weave_layer.W_PP,
+                                      0,
+                                      std=conv_weight_init_stddevs[ind])
+                nn.init.trunc_normal_(weave_layer.W_P,
+                                      0,
+                                      std=conv_weight_init_stddevs[ind])
+            self.layers.append(weave_layer)
+
+        self.dense1: nn.LazyLinear = nn.LazyLinear(self.n_graph_feat)
+        self.dense1_act = final_conv_activation_fn
+        self.dense1_bn: nn.BatchNorm1d = nn.BatchNorm1d(
+            num_features=self.n_graph_feat,
+            eps=1e-3,
+            momentum=0.99,
+            affine=True,
+            track_running_stats=True)
+
+        self.weave_gather = torch_layers.WeaveGather(
+            batch_size,
+            n_input=self.n_graph_feat,
+            gaussian_expand=gaussian_expand,
+            compress_post_gaussian_expansion=compress_post_gaussian_expansion)
+
+        if n_layers > 0:
+            self.layers2: nn.ModuleList = nn.ModuleList()
+            for ind, layer_size, weight_stddev, bias_const, dropout, activation_fn in zip(
+                [0, 1], fully_connected_layer_sizes, weight_init_stddevs,
+                    bias_init_consts, dropouts, self.activation_fns):
+                self.layer: nn.LazyLinear = nn.LazyLinear(layer_size)
+                self.layer.layer_bn = nn.BatchNorm1d(num_features=layer_size,
+                                                     eps=1e-3,
+                                                     momentum=0.99,
+                                                     affine=True,
+                                                     track_running_stats=True)
+                self.layer.weight_stddev = weight_stddev
+                self.layer.bias_const = bias_const
+                self.layer.dropout = nn.Dropout(dropout)
+                self.layer.layer_act = activation_fn
+                self.layers2.append(self.layer)
+
+        n_tasks = self.n_tasks
+        if self.mode == 'classification':
+            n_classes = self.n_classes
+            self.layer_2 = nn.LazyLinear(n_tasks * n_classes)
```

Comment:
@rbharath will it be fine to use lazy linear over default nn.linear here. The usage here is based on the fact that input size is not known, but it may be possible to get that.